### PR TITLE
Fix various issues with packager script

### PR DESCRIPTION
**Pull request by jrha**

### CODE REVIEW

File: src/releasing/packager.sh [hunk 1]
```diff
@@ -126,7 +127,7 @@ if gpg-agent; then
             echo "repository: $r" | sed 's/./=/g'
 
             echo "release tag: "
-            release_tag="$(git tag -l | grep -e "$RELEASE\$")"
+            release_tag="$(git tag -l | grep -e "$VERSION\$")"
```
Comment (wpoely86) [inline src/releasing/packager.sh:130]: `$VERSION` is something like `"7 (Core)"`. Are you sure that is want you want?

Comment (jrha) [reply src/releasing/packager.sh:130]: Ah yes, overloaded variable name, it's set on [lines 101-104](https://github.com/quattor/release/pull/346/files#diff-e6f5cbc97452a339c45695e32e8e7684fa4d8527da34df6760c8888c8217fdf1R101-R104).

Comment (jrha) [reply src/releasing/packager.sh:130]: I should probably choose a different name.

Comment (wpoely86) [reply src/releasing/packager.sh:130]: maybe pick QUATTOR_VERSION ?

**Review by wpoely86 — APPROVED** [review]

**Pull request merged**